### ISSUE TITLE
Add AniList <-> MAL links in anime/manga pages

### ISSUE THREAD

**Issue by darklinkpower** [MALSync/MALSync#472]
Having a link added would be useful to go from one site to another.

I think it would make sense to add the links above the external site links

![imagen](https://user-images.githubusercontent.com/1389286/93945557-c842ca80-fcfc-11ea-8386-0283ffdd6ba5.png)

![imagen](https://user-images.githubusercontent.com/1389286/93945638-f0cac480-fcfc-11ea-98bc-37feac37fb96.png)

**Comment by Karmesinrot** [comment]
There is already a similar feature under "Search", click on [show] then click on the respective site you'd want to search on.

**Comment by darklinkpower** [comment]
I'm aware of it, but with search it, well, opens a search url and you still need to pick the correct entry in the search, while this suggestion is to already put the correct link there.

**Comment by Karmesinrot** [comment]
That would require addtional entries to the database and those entries(along with site links) are checked manually hence it's difficult to maintain.

**Comment by darklinkpower** [comment]
Not really, the AniList has in it's database the MAL ids that correspond which each entry and its API can be used for matching. There are several examples of scripts that do this.

https://greasyfork.org/en/scripts/369792-maltoani
https://github.com/pilar6195/AniList-Extras

**Comment by henrik9999** [comment]
Yep @Karmesinrot our database has nothing to do with this

**Comment by Karmesinrot** [comment]
Ah I see.

**Comment by lolamtisch** [comment]
@darklinkpower 
With the new release we added this. But only for relations we could easily get.
Mal links to Anilist
Anilist links to Mal
Kitsu kinks to Mal and Anilist
Simkl links to Mal

I will keep this issue open as we plan add all relations relations someday in the future.

**Comment by darklinkpower** [comment]
Thank you very much, this is great!

**Comment by Pikcee** [comment]
Can we also add Mangaupdates link in AniList/MAL, it would be even much more useful than the MAL/AniList.
